vault/token_store.go

```diff
@@ -236,6 +254,12 @@ func (ts *TokenStore) paths() []*framework.Path {
 		{
 			Pattern: "create/" + framework.GenericNameRegex("role_name"),
 
+			DisplayAttrs: &framework.DisplayAttributes{
+				OperationPrefix: operationPrefixToken,
+				OperationVerb:   "create",
+				OperationSuffix: "against-role",
```

Comment:
should this be `with-role`

Reply:
I was basing this on the name of the request handler:

https://github.com/hashicorp/vault/blob/39ab2efb2d276fd5cb2af56697f8256ccfa4aba2/vault/token_store.go#L327-L329